app/views/templates/_help_accordion.html.erb

```diff
@@ -0,0 +1,16 @@
+          <div class="panel panel-default">
```

Comment:
Nitpick, lose the indentation, I don't think we normally indent fragments to match where they're called from, it's easier to edit (even if it renders without indentation).